app/models/legislator.rb

```diff
@@ -90,9 +90,10 @@ def self.recheck_reps_with_us
     end
   end
 
-  def self.find_or_create_by_hash(hash)
-    bioguide_id = hash.delete('bioguide_id')
-    create_with(hash).find_or_create_by(bioguide_id: bioguide_id)
+  def self.create_or_update(legislator_hash)
+    hash = legislator_hash.symbolize_keys
+    bioguide_id = hash.delete(:bioguide_id)
+    find_or_initialize_by(bioguide_id: bioguide_id).tap{|l| l.update(hash)}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.